herd/AArch64ASLSem.ml

```diff
@@ -816,11 +816,11 @@ module Make (TopConf : AArch64Sig.Config) (V : Value.AArch64ASL) :
       | V.Val cst -> (tr_cst (ASLScalar.convert_to_bv sz) cst, [])
 
     let aarch64_to_asl = function
-      | V.Var v -> ASLS.A.V.Var v
+      | V.Var (v, _) -> ASLS.A.V.Var (v, ASLS.A.V.SData.default)
```

Comment:
We may rather aim at keeping the symbolic data identical in both worlds:
```suggestion
      | V.Var (v, s) -> ASLS.A.V.Var (v, d)
```
This would mean exposing some type equality for symbolic data in both "V" modules.